src/components/apiErrorAlert/ApiErrorAlert.tsx

```diff
@@ -2,24 +2,27 @@ import { Alert } from '@navikt/ds-react';
 import React from 'react';
 
 import { ApiError } from '~api/apiClient';
+import { RevurderingErrorCodes } from '~components/apiErrorAlert/revurderingApiError/RevurderingApiError';
+import { FeilresponsErrorCodes, Generell } from '~components/apiErrorAlert/saksbehandlerApiError/ApiErrorTypes';
+import { SøknadsbehandlingErrorCodes } from '~components/apiErrorAlert/søknadsbehandlingApiError/SøknadsbehandlingApiError';
 import { useI18n } from '~lib/i18n';
 
-import messages from './ApiErrorAlert-nb';
 import styles from './apierroralert.module.less';
-import { feilresponsTilFeilmelding } from './ApiErrorAlertUtils';
 import revurderingsMessages from './revurderingApiError/RevurderingApiError-nb';
+import apiErrorMessages from './saksbehandlerApiError/ApiErrorAlert-nb';
 import søknadsbehandlingMessages from './søknadsbehandlingApiError/søknadsbehandlingApiError-nb';
 
-const ApiErrorAlert = (props: { error?: ApiError }) => {
+const ApiErrorAlert = ({
+    error,
+}: {
+    error?: ApiError<FeilresponsErrorCodes | RevurderingErrorCodes | SøknadsbehandlingErrorCodes | string>;
```

Comment:
blir det ikke litt 'svakere' typet nå när error-koden bara kan vara en string också?

Reply:
Jo, og jeg skulle ønske vi ikke trengte å ha string. Men "alle" steder hvor `ApiErrorAlert` er tatt i bruk, så er ikke errorObjektet typet med disse typede feilmeldingskodene (men default til `string`). Det hadde blitt så mange endringer om jeg skulle fikset opp i det (komponenten er vel brukt rundt 70 ganger mener jeg å huske). Så tenkte å legge opp en PR først for å se om folk var enige før vi evt fikser typene der og ;)

Så alt i alt er den _mer_ typesikret nå (om vi fjerner string)